docs/runtimes/configuring-job-resources.md

```diff
@@ -37,73 +37,72 @@ fn.set_envs(file_path="env.txt")
 
 ## Replicas
 
-Some runtimes can scale horizontally, configured either as a number of replicas:</br>
-`spec.replicas` </br>
-or a range (for auto scaling in Dask or Nuclio):</br>
+Some runtimes can scale horizontally, configured either as a number of replicas:
+```python
+training_function = mlrun.code_to_function("training.py", name="training", handler="train", 
+                                           kind="mpijob", image="mlrun/mlrun-gpu")
+training_function.spec.replicas = 2
 ```
-spec.min_replicas = 1
-spec.max_replicas = 4
+or a range (for auto-scaling in Dask or Nuclio):
+```python
+training_function = mlrun.code_to_function("training.py", name="training", handler="train", 
+                                           kind="mpijob", image="mlrun/mlrun-gpu")
+training_function.min_replicas = 1
+training_function.max_replicas = 4
```

Comment:
this says dask or nuclio but kind is mpijob

Reply:
changed example